cfgmgr/natmgr.cpp

```diff
@@ -2005,6 +2121,10 @@ void NatMgr::addStaticSingleNaptEntry(const string &key)
         FieldValueTuple s(TRANSLATED_L4_PORT, keys[2]);
         fvVectorDnat.push_back(r);
         fvVectorDnat.push_back(s);
+
+        /* Add DnatPool entry to APPL_DB*/
+        SWSS_LOG_INFO("Adding dnat pool entry for %s", m_staticNaptEntry[key].local_ip.c_str());
+        addDnatPoolEntry(m_staticNaptEntry[key].local_ip);
```

Comment:
Why do we need DNAT pool entry for the local_ip? I believe DNAT pool lookup is required only for the translated global ip before DNAT/DNAPT table lookup.

Reply:
User can configure the local IP as the translated source IP in the SNAT direction. Hence in the reverse DNAT direction, we need the DNAT Pool entry for the local IP.

Reply:
@kperumalbfn, wondering if this PR looks OK? Thanks.